<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="dodajDete">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>5</height>
     </size>
    </property>
    <property name="text">
     <string>primeni</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="andI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>5</height>
     </size>
    </property>
    <property name="text">
     <string>andI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="andE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>5</height>
     </size>
    </property>
    <property name="text">
     <string>andE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orI">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>5</height>
     </size>
    </property>
    <property name="text">
     <string>orI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orE">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>5</height>
     </size>
    </property>
    <property name="text">
     <string>orE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ponisti">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ponisti</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
